crates/resolver/src/name_server/name_server.rs

```diff
@@ -212,35 +209,43 @@ impl<C: DnsHandle<Error = ResolveError>, P: ConnectionProvider<Conn = C>> Ord fo
     }
 }
 
-impl<C: DnsHandle<Error = ResolveError>, P: ConnectionProvider<Conn = C>> PartialOrd
-    for NameServer<C, P>
+impl<C, P> PartialOrd for AbstractNameServer<C, P>
+where
+    C: DnsHandle<Error = ResolveError> + Send + Sync + 'static + CreateConnection,
+    P: RuntimeProvider + Send,
```

Comment:
In general, please don't do stylistic changes (like moving bounds from the `impl` to a `where` clause) at the same time as semantic changes, this makes it harder to review the code.

Reply:
Sorry for that. I just thought this change will make code clear and won't cause some confusion.